templates/section.html.slim

```diff
@@ -57,8 +57,9 @@
 
       - unless hide_title
         h2=section_title
-      - (blocks - vertical_slides).each do |block|
-        =block.convert
+      div.slide-content
+        - (blocks - vertical_slides).each do |block|
+          =block.convert
     - vertical_slides.each do |subsection|
```

Comment:
I'm a bit confused with this part of the code. I need/want to wrap the content of the slide (everything except the title in a `<div class="slide-content">`.

Reply:
The way I see it: vertical slides are just an organization mechanism. They are full slides in their own right so the `slide-content` div should apply to them individually, not to the vertical set. You seem to be doing the right thing here.